bindings/pyroot_experimental/PyROOT/src/PyROOTWrapper.cxx

```diff
@@ -0,0 +1,47 @@
+
+// Bindings
+#include "PyROOTWrapper.h"
+
+// Cppyy
+#include "CPyCppyy.h"
+#include "ProxyWrappers.h"
+
+// ROOT
+#include "TROOT.h"
+#include "TSystem.h"
+#include "TClass.h"
+#include "TDataMember.h"
+#include "TClassEdit.h"
+#include "TEnum.h"
+#include "TEnumConstant.h"
+#include "TInterpreter.h"
+#include "TGlobal.h"
+#include "DllImport.h"
+
+namespace PyROOT {
+R__EXTERN PyObject *gRootModule;
+}
+
+using namespace PyROOT;
+
+namespace {
+
+inline void AddToGlobalScope(const char *label, const char * /* hdr */, TObject *obj, Cppyy::TCppType_t klass)
```

Comment:
No need to mark this `inline`. But declare it as `static` to guarantee local linkage.